pkg/controller/validation/account_validation_controller.go

```diff
@@ -202,10 +203,8 @@ func ValidateAccountTags(client awsclient.Client, accountId *string, shardName s
 
 					return nil
 				} else {
-					return &AccountValidationError{
-						Type: IncorrectOwnerTag,
-						Err:  fmt.Errorf("Account is not tagged with the correct owner, has %s; want %s", *tag.Value, shardName),
-					}
+					log.Info(fmt.Sprintf("Account is not tagged with the correct owner, has %s; want %s", *tag.Value, shardName))
```

Comment:
I removed the error here as it was really just polluting the logs and causing the validation controller to continually retry when the flag was turned off, which is just wasting compute cycles when the flag is off.

Reply:
I'm surprised this didn't break any tests, as I thought tests here were validating for a specific error type with a correct message.  Either way I'm fine with it.